third_party/mesa/src/compiler/isaspec/isaspec_decode_impl.c

```diff
@@ -184,7 +184,30 @@ isa_print(struct isa_print_state *state, const char *fmt, ...)
 	int ret;
 
 	va_start(args, fmt);
-	ret = vasprintf(&buffer, fmt, args);
+// GOOGLE: vasprintf is not directly available in Windows.
```

Comment:
similar to `strndup`, `vasprintf` could probably moved to the dive specific .h/.c file